lib/rucio/transfertool/fts3.py

```diff
@@ -12,25 +12,30 @@
 # - Martin Barisits, <martin.barisits@cern.ch>, 2017-2018
 # - Eric Vaandering, <ewv@fnal.gov>, 2018
 # - Diego Ciangottini <diego.ciangottini@pg.infn.it>, 2018
+# - Cedric Serfon <cedric.serfon@cern.ch>, 2018
 
 from __future__ import absolute_import
 import datetime
 import json
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
+    JSONDecodeError = ValueError
 import logging
 import sys
 import time
 import urlparse
 import uuid
-import traceback
 
 import requests
+from requests.adapters import ReadTimeout
 from requests.packages.urllib3 import disable_warnings  # pylint: disable=import-error
 
 from dogpile.cache import make_region
 from dogpile.cache.api import NoValue
 
-from fts3.rest.client.easy import Context, delegate
-from fts3.rest.client.exceptions import BadEndpoint, ClientError, ServerError
+from fts3.rest.client.easy import Context, delegate  # pylint: disable=no-name-in-module,import-error
+from fts3.rest.client.exceptions import BadEndpoint, ClientError, ServerError, TransferToolTimeout, TransferToolWrongAnswer  # pylint: disable=no-name-in-module,import-error
```

Comment:
These imports are wrong.
TransferToolTimeout, TransferToolWrongAnswer needs to be imported from rucio.common.exception